### ISSUE TITLE
Impossible to configure fuse in NixOS

### ISSUE THREAD

**Issue by kmarisa** [NixOS/nixpkgs#59021]
## Issue description
It is impossible to configure fuse in 18.09 because fuse looks in the nix store for the fuse.conf file and not in ```/etc```, so ```environment.etc."fuse.conf"``` doesn't work anymore. As a result, I am unable to set the ```user_allow_other``` option, which breaks my sshfs setup.

## Technical details
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.
```
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.30, NixOS, 18.09.2436.395a543f360 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(marisa): `"nixos-18.09.1819.76aafbf4bf4, nixos-unstable-19.03pre168555.ce8c2436998, nixpkgs-unstable-19.03pre168583.0710c9fc25b"`
 - channels(alice): `""`
 - channels(root): `"nixos-18.09.2436.395a543f360, unstable-19.09pre173349.07b42ccf2de"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by furrycatherder** [comment]
https://github.com/NixOS/nixpkgs/pull/53074

**Comment by matthewbauer** [comment]
Usually this just requires setting ```sysconfdir=/etc``` or something like that

**Comment by kmarisa** [comment]
Where do I set that?

**Comment by matthewbauer** [comment]
This is Fuse 3 right? Could you verify this fixes the issue: #59043 - I can't remember how to use fusermount
